API/CLI.md

```diff
@@ -15,6 +15,20 @@ The best place to go next is [the Tessel 2 start experience](http://tessel.io/t2
 ## Updating Tessel 2 On-board OS/Firmware
 
 ### How do I know if I need to update my T2?
+T2 - CLI and T2 - Firmware are separately versioned.
```

Comment:
This spacing makes it hard to read, and it won't be control-f search friendly for users that would spell it out as it appears in the respective projects, ie: "t2-cli" and "t2-firmware"